nagios/datadog_checks/nagios/nagios.py

```diff
@@ -64,10 +73,20 @@ class Nagios(AgentCheck):
         re.compile('^(?P<key>service_perfdata_file)\s*=\s*(?P<value>.+)$'),
     ]
 
+    def gauge(self, *args, **kwargs):
+        """
+        Compatability wrapper for Agents that do not submit gauge metrics with custom timestamps
+        """
+        if 'timestamp' in kwargs and 'timestamp' not in getargspec(super(Nagios, self).gauge).args:
+            del kwargs['timestamp']
+
+        super(Nagios, self).gauge(*args, **kwargs)
```

Comment:
we can spare a call to `super` if you assign it at the beginning of the function body